safe/definitions/exposure_classifications.py

```diff
@@ -531,6 +553,28 @@
                 }
             ]
         },
+        {
+            # IF YOU UPDATE THIS ONE, PLEASE UPDATE THE OTHER ONE IN BUILDINGS
+            'key': 'evacuation centre',
+            'name': tr('Evacuation centre'),
+            'description': tr(
+                'Evacuation centres provide a place of temporary refuge for '
+                'people evacuated from homes in the vicinity of a disaster '
+                'and for people who may be travelling through the affected '
+                'area but are unable to continue.'),
+            'string_defaults': [
+                'evacuation centre',
+                'refugee',
+                'idp camp'
+            ],
+            'osm_downloader': [],
+            'citations': [
+                {
```

Comment:
If this definition is from a specific source, please cite the source

Reply:
I know it was weird, but I found this definition on the google page. Then when I clicked the link, the definition wasn't existing anymore on the website. I tried this morning, my history navigation, still the same, I can't find the website again.